frontstage/templates/secure-messages/secure-messages-view.html

```diff
@@ -1,82 +0,0 @@
-{% extends "secure-messages/messages.html" %}
```

Comment:
This wasn't been used and so has been removed, there are a few of these and associated tests